frontend/src/components/Footer/Footer.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Footer from './Footer';
+
+import { render } from '@testing-library/react';
+import "@testing-library/jest-dom/extend-expect"; 
+
+it("renders without crashing", ()=> {
+    const div = document.createElement("div");
+    ReactDOM.render(<Footer />, div);
+});
+
+it("renders Footer correctly", ()=> {
+    const {getByTestId} = render(<Footer text="Copyright © TourDeFoos 2021"/>)
+    expect(getByTestId('footer')).toHaveTextContent("Copyright © TourDeFoos 2021")
+});
+
+it("renders Footer correctly", ()=> {
```

Comment:
Dwa testy z tym samym opisem? Jakoś trzeba rozróżnić przypadek testowy, albo zrobić test parametryzowany. Z tego co kojarzę nawet przy puszczeniu wszystkich będzie odpalał się tylko jeden z nich.

Reply:
Racja, zmieniam to na:
1) "renders Footer correctly with default text"
2) "renders Footer correctly with given text"

Dzięki!